doc/source/scans.rst

```diff
@@ -9,19 +9,19 @@ parameter settings (e.g. related to :ref:`geometry <sbgeo>` and :ref:`particle
 sources <sbparticlegen>`), before final conclusions can be drawn. In a simple
 case one might for instance, want to plot the value of some figure-of-merit as a
 function of a parameter (i.e. the thickness of some geometry volume, or the
-enrichment level of a particular material).  Unfortunately, launching the
+enrichment level of a particular material). Unfortunately, launching the
 relevant simulation and analysis jobs manually, while keeping track of both
 input parameters and output files, can be both tedious and error prone. To
 facilitate this process, dgcode provides
 :sbpkg:`ScanLauncher<ScanUtils/python/ScanLauncher.py>` and
 :sbpkg:`ScanLoader<ScanUtils/python/ScanLoader.py>` utility classes in the
-:sbpkg:`ScanUtils` package. These utilities assume that the 
+:sbpkg:`ScanUtils` package. These utilities assume that the...FIXME
```

Comment:
Right. Just leave the FIXME there for now, I will look at it later then.

Reply:
Actually, the `These utilities assume that the...FIXME` can simply be removed, I don't think the sentence is needed at all anymore.